packages/python-packages/api-stub-generator/apistub/nodes/_class_node.py

```diff
@@ -117,56 +134,53 @@ def _inspect(self):
         # Find any ivar from docstring
         self._parse_ivars()
 
+        is_typeddict = hasattr(self.obj, "__required_keys__") or hasattr(self.obj, "__optional_keys__")
+
         # find members in node
         for name, child_obj in inspect.getmembers(self.obj):
             if inspect.isbuiltin(child_obj):
                 continue
             elif self._should_include_function(child_obj):
                 # Include dunder and public methods
                 if not name.startswith("_") or name.startswith("__"):
-                    self.child_nodes.append(
-                        FunctionNode(self.namespace, self, child_obj)
-                    )
-            elif self.is_enum and isinstance(child_obj, self.obj):
+                    try:
+                        self.child_nodes.append(
+                            FunctionNode(self.namespace, self, child_obj)
+                        )
+                    except OSError:
+                        # Don't create entries for things that don't have source
+                        pass
+            elif name == "__annotations__":
+                for (item_name, item_type) in child_obj.items():
+                    if item_name.startswith("_"):
+                        continue
+                    if is_typeddict and inspect.isclass(item_type) or getattr(item_type, "__module__", None) == "typing":
```

Comment:
I assume that this will work 'cause we are python 3.7+ (i.e. `TypedDict` is not from `typing_extensions`).

Reply:
![image](https://user-images.githubusercontent.com/5723682/154116822-68114e7e-9149-4bba-a416-bb0e4fe0df70.png)
It's not because TypedDict is from typing but because the key type could be from `typing`.